inspirehep/modules/migrator/tasks.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 #
 # This file is part of INSPIRE.
-# Copyright (C) 2015 CERN.
+# Copyright (C) 2014, 2015, 2016 CERN.
```

Comment:
You don't need to put this change into your commit since you don't touch this file.